calculator.py

```diff
@@ -72,3 +66,4 @@ def prime_no(n) :
     else :
         print("Invalid Input")
 print("EXIT")
+

```

Comment:
I think is necesary to create the main 
Also a diferent way to call the operation, it could be by the use of functions